crates/store/re_types/definitions/rerun/blueprint/components/timeline.fbs

```diff
@@ -0,0 +1,14 @@
+
+namespace rerun.blueprint.components;
+
+/// A timeline, identified by its name.
+table Timeline (
```

Comment:
I think we should call this `TimelineName` for consistency.

A timeline is currently defined by its name together with its type:

https://github.com/rerun-io/rerun/blob/d3ad64e919e76b76688f53e923d650a31d813108/crates/store/re_log_types/src/time_point/timeline.rs#L22-L28

See also:
* https://github.com/rerun-io/rerun/issues/4080